mirdata/download_utils.py

```diff
@@ -71,14 +75,27 @@ def downloader(
     if not os.path.exists(save_dir):
         os.makedirs(save_dir)
 
+    if not index:
+        raise ValueError("Index must be specified.")
+
     if cleanup:
         logging.warning(
             "Zip and tar files will be deleted after they are uncompressed. "
             + "If you download this dataset again, it will overwrite existing files, even if force_overwrite=False"
         )
 
-    if remotes is not None:
-        if partial_download is not None:
+    if index.remote:
```

Comment:
I'm sorry I think I'm being slow, but I'm confused by which this index is, because the `indexes` under `datasets/indexes` are unchanged right? And they don't have a `remote` key but neither do the `INDEXES` dictionary in the loader? Am I completely lost?

Reply:
Right - the indexes under datasets/indexes are unchanged.
Maybe the confusing thing is that there's `INDEXES` which is a dictionary, and the values are `core.Index` objects. The input to the downloader is a single `core.Index` object, not `INDEXES`. `The `Index` objects all have `.remote` attributes, but most of them are set to `None`. Right now only acousticbrainz_genre has an index with .remote set.